### PR TITLE
sync-for-science/tracking#121 - More override changes.

### PR DESCRIPTION
**Pull request by mtmcduffie**

### CODE REVIEW

**Comment by mtmcduffie** [comment]
New PR for requested changes.

**Comment by erikwiffin** [comment]
Looks good, thanks! Sorry for all the back and forth.

File: auth_proxy/proxy/flask.py [hunk 1]
```diff
@@ -46,43 +46,35 @@ def request(self):
...
-        :return: list, dict : The modified request arguments and headers.
+        Strip existing security arguments and apply our own.
+        :param original_args: 
+        :return: list: new request arguments with injected security
         """
-
-        original_args = list(self.orig.args.items())
-
-        # Strip existing security arguments so we can apply our own.
         stripped_args = [arg for arg in original_args if arg[0] != self.SECURITY_ARG_NAME]
```
Comment (jmandel) [inline auth_proxy/proxy/flask.py]: by the time you're here, you can just:

```
return [arg for arg in original_args if arg[0] != self.SECURITY_ARG_NAME] + \
    [self.SECURITY_ARG_NAME, self._get_scope_security_label()] + \
    [self.SECURITY_ARG_NAME, self._get_patient_security_label()]
```

and avoid even the function-internal notation. (But some might just accuse me of playing code golf here ;-))

Comment (mtmcduffie) [reply auth_proxy/proxy/flask.py]: Duly noted! I'm a glutton for new lines when I think it improves my internal version of readability.

**Pull request merged**